thirdeye/thirdeye-frontend/app/pods/components/rootcause-chart-toolbar/template.hbs

```diff
@@ -0,0 +1,49 @@
+<div class="col-xs-2">
+  {{date-range-picker
+    class="te-range-picker"
+    start=analysisRangeStart
+    end=analysisRangeEnd
+    maxDate=analysisRangeMax
+    ranges=analysisRangePredefined
+    showCustomRangeLabel=false
+    format="MMM D"
+    serverFormat="YYYY-MM-DD HH:mm"
+    applyAction=(action "onAnalysisRange")
+  }}
+</div>
+<div class="col-xs-2">
+  {{#power-select
+    selected=granularity
+    options=granularityOptions
+    searchEnabled=false
+    triggerId="select-granularity"
+    onchange=(action "onGranularity")
+  as |granularity|
+  }}
+    {{granularity}}
+  {{/power-select}}
+</div>
+<div class="col-xs-2">
+  {{#power-select
+    selected=compareMode
+    options=compareModeOptions
+    searchEnabled=false
+    triggerId="select-compare-mode"
+    onchange=(action "onCompareMode")
+  as |mode|
```

Comment:
This needs to be indented by 1 more space

Reply:
fixed